internal/performance/export_test.go

```diff
@@ -35,28 +35,43 @@ import (
 	"github.com/google/exposure-notifications-server/pkg/base64util"
 	"github.com/google/exposure-notifications-server/pkg/util"
 	pgx "github.com/jackc/pgx/v4"
+	"github.com/sethvargo/go-envconfig"
 	"github.com/sethvargo/go-retry"
 )
 
+type testConfig struct {
+	Publishes int  `env:"PUBLISHES,required"`
```

Comment:
FYI defaults can be set here

Reply:
good to know! will update tomorrow morning